// [[Reason: Sourcemap is more detailed]] ////

//// [tests/cases/compiler/declarationEmitDuplicateParameterDestructuring.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,12 +1,12 @@
 
 
 //// [declarationEmitDuplicateParameterDestructuring.d.ts]
-export declare const fn1: ({ prop: a, prop: b }: {
+export declare const fn1: ({ prop, prop }: {
     prop: number;
 }) => number;
-export declare const fn2: ({ prop: a }: {
+export declare const fn2: ({ prop }: {
     prop: number;
-}, { prop: b }: {
+}, { prop }: {
     prop: number;
 }) => number;
 //# sourceMappingURL=declarationEmitDuplicateParameterDestructuring.d.ts.map
\ No newline at end of file
